index.html

```diff
@@ -50,5 +50,223 @@ <h1>Hey There. I'm Carlyle</h1>
+                  <li class="tag bootag cardtagbg">bootstrap</li>
+                  <li class="tag rubytag cardtagbg">Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+           </div>
+            <div class="card card5">
+              <div class="card-background">
+                <div class="bcg-settings"></div>
+                <div class="img-placeholder"></div>
+              </div>
+              <div class="card-content">
+                <div class="cardtitle">
+                  <h3 class="ct-text">Profesional Art Printing Data</h3>
+                </div>
+                <div class="cardagraph">
+                  <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                </div>
+                <div class="cardtags">
+                  <ul class="tags">
+                    <li class="tag htag cardtagbg">html</li>
+                    <li class="tag bootag cardtagbg">bootstrap</li>
+                    <li class="tag rubytag cardtagbg">Ruby</li>
+                  </ul>
+                </div>
+              </div>
+              <div class="fullwidth">
+                <button type="button" class="probtn1">See Project</button>
+              </div>
+            </div>
+            <div class="card card6">
+              <div class="card-background">
+                <div class="bcg-settings"></div>
+                <div class="img-placeholder"></div>
+              </div>
+              <div class="card-content">
+                <div class="cardtitle">
+                  <h3 class="ct-text">Profesional Art Printing Data</h3>
+                </div>
+                <div class="cardagraph">
+                  <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+                </div>
+                <div class="cardtags">
+                  <ul class="tags">
+                    <li class="tag htag cardtagbg">html</li>
+                    <li class="tag bootag cardtagbg">bootstrap</li>
+                    <li class="tag rubytag cardtagbg">Ruby</li>
+                  </ul>
+                </div>
+              </div>
+              <div class="fullwidth">
+                <button type="button" class="probtn1">See Project</button>
+              </div>
+            </div>
+          </div>
+      </div>
+    </section>
+    <section class="info">
+      <div class="about">
+        <h2 class="am-title">About Me</h2>
+        <p class="am-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] **As much as possible, your design should be a clone/similar to the Figma design. Let's implement the visual changes pointed out below.

**Figma Sample**
![image](https://user-images.githubusercontent.com/27709832/148975217-9a9983fb-0bc6-4068-83fa-8c32ea8f3bf9.png)

**Your Design**
![image](https://user-images.githubusercontent.com/27709832/148975444-17e44dbc-5680-4952-86c3-fff1927b138f.png)

Reply:
I worked on the second figma template, not the first
Please check again, you're getting me confused.